app/models/program.rb

```diff
@@ -49,7 +51,7 @@ def confirmed
 #   validates :conference_id, presence: true, uniqueness: true
   validates :rating, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 10 }
 
-  before_create :create_event_types
+  after_create :create_event_types
```

Comment:
@differentreality On creating a conference, it creates these 2 event types as well.Do we need to show that in the changelog page.If not paper_trail tracking  of Event Type should be turned off inside `create_event_types`.What do you think?